### PR TITLE
test: add unit test for aggregator two level merge

### PR DESCRIPTION
**Pull request by SeaRise**
### What problem does this PR solve?

Issue Number: ref #5900 

Problem Summary:

### Check List

Tests 

- [ ] Unit test
- [ ] Integration test
- [ ] Manual test (add detailed scripts or steps below)
- [ ] No code

Side effects

- [ ] Performance regression: Consumes more CPU
- [ ] Performance regression: Consumes more Memory
- [ ] Breaking backward compatibility

Documentation

- [ ] Affects user behaviors
- [ ] Contains syntax changes
- [ ] Contains variable changes
- [ ] Contains experimental features
- [ ] Changes MySQL compatibility

### Release note

```release-note
None
```

### CODE REVIEW

File: dbms/src/Flash/tests/gtest_aggregation_executor.cpp [hunk 1]
```diff
@@ -510,8 +557,27 @@ try
...
+    std::vector<String> tables{"big_table_1", "big_table_2", "big_table_3"};
+    for (const auto & table : tables)
+    {
+        auto request = context
+                           .scan("test_db", table)
+                           .aggregation({Max(col("value"))}, {col("key")})
+                           .build(context);
+        auto expect = executeStreams(request, 1);
+        // test for one level merge and two level merge.
+        std::vector<UInt64> two_level_thresholds{0, 1, 100000, std::numeric_limits<UInt64>::max()};
```
Comment (xzhangxian1008) [inline dbms/src/Flash/tests/gtest_aggregation_executor.cpp:572]: We only have 200 keys, are 100000 and `std::numeric_limits<UInt64>::max()` necessary?

Comment (SeaRise) [reply dbms/src/Flash/tests/gtest_aggregation_executor.cpp:572]: yes, you are right.
100000 and `std::numeric_limits<UInt64>::max()` are useless.

Comment (SeaRise) [reply dbms/src/Flash/tests/gtest_aggregation_executor.cpp]: updated.

**Comment by xzhangxian1008** [comment]
lgtm

**Comment by SeaRise** [comment]
merged to https://github.com/pingcap/tiflash/pull/6549

**Pull request closed without merging**